### PR TITLE
fix: redirect apt output to /dev/null

### PR DESCRIPTION
**Pull request by allan-bailey**
So the `-qq` option doesn't work, so adding redirect to /dev/null

### CODE REVIEW

File: Docker/Dockerfile [hunk 1]
```diff
@@ -11,7 +11,7 @@ WORKDIR "$WORKDIR/narwhal"
 ARG BUILD_MODE
 
 # Install basic dependencies
-RUN apt-get -qq update \
+RUN DEBIAN_FRONTEND=noninteractive apt-get -qq update \
```
Comment (akichidis) [inline Docker/Dockerfile:14]: If the `-qq` doesn't work do we still want to leave it there?

Comment (allan-bailey) [reply Docker/Dockerfile:14]: For `apt update` it does work, for `install` it doesn't.

File: Docker/Dockerfile [hunk 2]
```diff
@@ -21,7 +21,7 @@ RUN apt-get -qq update \
     libssl-dev \
     pkg-config \
     clang \
-    cmake \
+    cmake  2> /dev/null > /dev/null \
```
Comment (huitseeker) [inline Docker/Dockerfile:24]: Would `> /dev/null 2>&1` not work in a more readable fashion?

**Review by huitseeker — APPROVED** [review]
Left a comment for an example.

**Pull request merged**